plugins/outputs/timestream/timestream.go

```diff
@@ -322,25 +322,25 @@ func (t *Timestream) writeToTimestream(writeRecordsInput *timestreamwrite.WriteR
 }
 
 func (t *Timestream) logWriteToTimestreamError(err error, tableName *string) {
-	t.Log.Errorf("Failed to write to Timestream database '%s' table '%s'. Skipping metric! Error: '%s'",
+	t.Log.Errorf("Failed to write to Timestream database %q table %q: %w. Skipping metric!",
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.